python/py-pdbx/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-pdbx
+version             2.0.1
+revision            0
+
+checksums           rmd160  68a545db541a66af24eece3a60050be437199b00 \
+                    sha256  ad64a02a59d41e0013554ba03153d2ecbcb55579453b49be89c87ed228d0d1b1 \
+                    size    2149838
+
+supported_archs     noarch
+platforms           {darwin any}
+license             CCO-1.0
+maintainers         {gmail.com:howarth.at.macports @jwhowarth} openmaintainer
+
+description         Utilities for PDBx/mmCIF storage model
+long_description    {*}${description}
+
+github.setup            Electrostatics    mmcif-pdbx    ${version} v
+github.tarball_from     archive
+master_sites            https://github.com/Electrostatics/mmcif_pdbx/archive/refs/tags/v${version}:mmcif_pdbx
+distfiles               mmcif_pdbx-v${version}.tar.gz:mmcif_pdbx
```

Comment:
both `master_sites` and `distfiles` will be the default and these lines can thus be removed if you correct your mistake above in the `github.setup` line